src/doom/d_main.c

```diff
@@ -1599,6 +1599,7 @@ void D_DoomMain (void)
     if (!M_ParmExists("-noautoload") && gamemode != shareware)
     {
         char *autoload_dir;
+        DEH_printf("I_Init: Autoloading.\n");
```

Comment:
Logging changes in their own separate commit.

Reply:
If doom.exe didn't print these lines under DOS, then it's entirely unnecessary to add them here.

Reply:
I think the logging would be useful for the next unfortunate soul who has to debug early init failure